crafts.lua

```diff
@@ -1,3 +1,59 @@
+local materials = {
+    dirt = "default:dirt",
+    sand = "default:sand",
+    gravel = "default:gravel",
+    copper_ingot = "default:copper_ingot",
+    steel_ingot = "default:steel_ingot",
+    gold_ingot = "default:gold_ingot",
+    tin_ingot = "default:tin_ingot",
+    mese_crystal_fragment = "default:mese_crystal_fragment",
+    torch = "default:torch",
+    diamond = "default:diamond",
+    clay_lump = "default:clay_lump",
+    water_bucket = "bucket:bucket_water",
+    empty_bucket = "bucket:bucket_empty",
+    dye_dark_grey = "dye:dark_grey",
+}
+
+if minetest.get_modpath("mcl_core") then
+    materials = {
+        dirt = "mcl_core:dirt",
+        sand = "mcl_core:sand",
+        gravel = "mcl_core:gravel",
+        steel_ingot = "mcl_core:iron_ingot",
+        gold_ingot = "mcl_core:gold_ingot",
+        mese_crystal_fragment = "mesecons:redstone",
+        torch = "mcl_torches:torch",
+        diamond = "mcl_core:diamond",
+        clay_lump = "mcl_core:clay_lump",
+        water_bucket = "mcl_buckets:bucket_water",
+        empty_bucket = "mcl_buckets:bucket_empty",
+        dye_dark_grey = "mcl_dye:dark_grey",
+        --use iron where no equivalent
+        copper_ingot = "mcl_core:iron_ingot",
+        tin_ingot = "mcl_core:iron_ingot",
+        silver_ingot = "mcl_core:iron_ingot",
+    }
+elseif minetest.get_modpath("fl_ores") and minetest.get_modpath("fl_stone") then
+    materials = {
+        dirt = "fl_topsoil:dirt",
+        sand = "fl_stone:sand",
+        gravel = "fl_topsoil:gravel",
+        steel_ingot = "fl_ores:iron_ingot",
+        gold_ingot = "fl_ores:gold_ingot",
+        mese_crystal_fragment = "fl_ores:iron_ingot",
+        torch = "fl_light_sources:torch",
+        diamond = "fl_ores:diamond",
+        clay_lump = "fl_bricks:clay_lump",
+        water_bucket = "fl_liquids:water_source",
+        empty_bucket = "air",
```

Comment:
Is this `air` really correct item here? Or should it be empty item / no replacements at all and such?

Reply:
https://github.com/mt-mods/basic_materials/pull/8/commits/ab90fdc6c38270438ceaf1b1ad2f223269ef4ea4 forgot to save, it was from when i was checking replacements and i3